_ingest-pipelines/processors/split.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: default
+title: Split
+parent: Ingest processors
+nav_order: 270
+---
+
+# Split processor
+
+The `split` processor is used to split a string field into an array of substrings based on a specified delimiter.
+
+The following is the syntax for the `split` processor:
+
+```json
+{
+  "split": {
+    "field": "field_to_split",
+    "separator": "delimiter",
+    "target_field": "split_field"
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `split` processor.
+
+Parameter | Required/Optional | Description |
+|-----------|-----------|-----------|
+`field` | Required | The field containing the string to be split. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets).
+`separator` | Required | The delimiter used to split the string. This can be a regular expression pattern.
+`target_field` | Required | The field where the array of substrings is stored. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets).
```

Comment:
`target_field` is optional, defaults to `field` if not specified, and doesn't support template snippets.